runners/spark/src/main/java/org/apache/beam/runners/spark/SparkPipelineOptions.java

```diff
@@ -101,4 +101,24 @@ public String create(PipelineOptions options) {
   boolean getUsesProvidedSparkContext();
   void setUsesProvidedSparkContext(boolean value);
 
+  /**
+   * List of local files to make available to workers.
+   *
+   * <p>Jars are placed on the worker's classpath.
+   *
+   * <p>The default value is the list of jars from the main program's classpath.
+   */
+  @Description("Jar-Files to send to all workers and put on the classpath. "
+      + "The default value is all files from the classpath.")
+  @Default.InstanceFactory(EmptyPathList.class)
```

Comment:
I would drop the default since:
1) It will prevent people from doing pipelineOptions.as(SparkPipelineOptions.class).as(DataflowPipelineWorkerPoolOptions.class) since the PipelineOptionsFactory prevents users from attempting to bind multiple incompatible interfaces with each other and performing the null check vs isEmpty is trivial.
2) Also users may want to not submit any jars since they may already have their cluster configured with all the code and you won't be able to differentiate unset from empty.